functional-tests/src/test/java/helm/EFKHelmChartIT.java

```diff
@@ -334,7 +332,7 @@ public boolean isNamespaceReady(K8sCluster cluster, String sNamespace)
         {
         CapturingApplicationConsole console = new CapturingApplicationConsole();
 
-        Arguments args = Arguments.of("get", "configmap", "coherence-internal-config");
+        Arguments args = Arguments.of("get", "secret", "coherence-service-binding");
```

Comment:
I would call it `coherence-monitoring-config` instead of `coherence-service-binding`.